packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -338,6 +380,51 @@ export class MgtLogin extends MgtTemplatedComponent {
     );
   }
 
+  /**
+   * POC for multi accounts - temporary
+   *
+   * @return {*}
+   * @memberof MgtLogin
+   */
+  renderAccounts() {
+    if (Providers.globalProvider.state === ProviderState.SignedIn && !Providers.globalProvider.isMultiAccountDisabled) {
```

Comment:
I'm noticing `!Providers.globalProvider.isMultiAccountDisabled` makes the code harder to read. @amrutha95, what do you think about this:

1. We add a `isMultiAccountSupported` to IProvider for any provider to implement. The `isMultiAccountSupported` is a getter that returns false by default.
2. We keep `isMultiAccountDisabled` as a property on Msal2Provider to disable it. The Msal2Provider implementation of `isMultiAccountSupported would return `!this.isMultiAccountDisabled`.
3. mgt-login uses `provider.IsMultiAccountSupported` everywhere

Reply:
Sounds good to me